Dockerfile

```diff
@@ -35,8 +35,9 @@ RUN git checkout $MENDER_VERSION
 COPY debian-master debian-master
 COPY debian-2.1.x debian-2.1.x
 
-# And add systemd service file for the modern recipes
-RUN cp support/mender.service debian-master/mender-client.service
+# And add systemd service file for the recipes. It is called "mender" for 2.3.x and before
+RUN cp support/mender-client.service debian-master/mender-client.service || \
```

Comment:
Btw, if you want to be fancy:
`cp support/(mender|mender-client).service debian-master/mender-client.service`

Reply:
Really? Is that `bash` specific? I haven't seen that before...

Reply:
That'd be `bash` yeh. I guess most modern shells should mimic this though.